protocol-test-util/pom.xml

```diff
@@ -47,6 +47,11 @@
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.atomix</groupId>
+      <artifactId>atomix</artifactId>
+      <version>3.0.6</version>
```

Comment:
This version should be managed in the parent pom.xml. It should there already be configured so we do not have to specify the version in sub modules.